src/main/java/com/jagrosh/vortex/commands/tools/LookupCmd.java

```diff
@@ -190,45 +191,51 @@ private void lookupGuild(String inviteCode, CommandEvent event)
     }
     
-    private Message constructMessage(Invite invite, Widget widget)
+    private Message constructMessage(Invite invite, Widget widget, long input)
     {
-        String gname;
-        long gid;
-        int users;
+        String gname = null;
+        long gid = 0L;
+        int users = -1;
         if(invite == null)
         {
             if(widget == null)
-                return new MessageBuilder().append(Constants.ERROR + " No users, guilds, or invites found.").build();
-            else if (!widget.isAvailable())
-                return new MessageBuilder().append(Constants.SUCCESS + " Guild with ID `" + widget.getId() + "` found; no further information found.").build();
+            {
+                if(input == 0)
+                    return new MessageBuilder().append(Constants.ERROR + " No users, guilds, or invites found.").build();
+
+                return new MessageBuilder()
+                        .append(UNKNOWN_ID + " Information about an unknown ID:")
+                        .setEmbed(new EmbedBuilder()
+                                .appendDescription(LINESTART + "Creation: " + "**"+TimeUtil.getTimeCreated(input).format(DateTimeFormatter.RFC_1123_DATE_TIME)+"**").build())
+                        .build();
+            }
+
+
             gid = widget.getIdLong();
-            gname = widget.getName();
-            users = widget.getMembers().size();
+            if(widget.isAvailable())
+            {
+                gname = widget.getName();
+                users = widget.getMembers().size();
+            }
         }
         else
         {
             Invite.Guild g = invite.getGuild();
-            if(g == null)
-            {
-                gid = 0L;
-                gname = "Unknown Guild";
-                users = 0;
-            }
-            else
+            if(g != null)
             {
                 gid = g.getIdLong();
                 gname = g.getName();
                 users = g.getOnlineCount();
             }
         }
         
-        String text = GUILD_EMOJI + " Information about **" + gname + "**:";
+        String text = GUILD_EMOJI + " Information about " + (gname == null ? "an unknown guild" : "**"+gname+"**") + ":";
         EmbedBuilder eb = new EmbedBuilder();
-        eb.appendDescription(LINESTART + "ID: **" + gid + "**"
-                + "\n" + LINESTART + "Creation: **" + TimeUtil.getTimeCreated(gid).format(DateTimeFormatter.RFC_1123_DATE_TIME) + "**"
+        eb.appendDescription(LINESTART + "ID: " + (gid == 0 ? "N/A" : "**"+gid+"**")
+                + "\n" + LINESTART + "Creation: " + (gid == 0 ? "N/A" : "**"+TimeUtil.getTimeCreated(gid).format(DateTimeFormatter.RFC_1123_DATE_TIME)+"**")
                 + "\n" + LINESTART + "Users: " + (users == -1 ? "N/A" : "**" + users + "** online")
                 + (widget != null && widget.isAvailable() ? "\n" + LINESTART + "Channels: **" + widget.getVoiceChannels().size() + "** voice" : ""));
```

Comment:
For consistency with the Channels info, I should perhaps only show the user count when it's available.